common/flags.ts

```diff
@@ -9,6 +9,8 @@ const DEV = () => process.env.NODE_ENV !== 'production'
 const ON = () => true
 /* eslint-enable @typescript-eslint/no-unused-vars */
 
+/** Special error handling for local development */
+export const DEV_ERROR = DEV()
```

Comment:
Wasn't sure if it made sense to export `DEV()`

Reply:
This seems fine, yeah